boot.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/deis/registry-token-refresher/pkg"
+	"github.com/deis/registry-token-refresher/pkg/credentials"
+	"k8s.io/kubernetes/pkg/api"
+	kcl "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+var appList []string
```

Comment:
no...main method just updates it and goroutine just reads it and go routine can have slight old copy as it is used only during the update event.